app/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "drizzle": "^1.3.3",
-    "drizzle-react": "^1.2.0",
-    "drizzle-react-components": "^1.3.0",
+    "@drizzle/store": "^1.5.3",
```

Comment:
would any `import ... from { Drizzle }` statements need to change in the source files now that this dependency has a new name?